src/ol/format/wmtscapabilitiesformat.js

```diff
@@ -0,0 +1,400 @@
+goog.provide('ol.format.WMTSCapabilities');
+
+goog.require('goog.asserts');
+goog.require('goog.dom.NodeType');
+goog.require('goog.object');
+goog.require('goog.string');
+goog.require('ol.extent');
+goog.require('ol.format.OWS');
+goog.require('ol.format.XLink');
+goog.require('ol.format.XML');
+goog.require('ol.format.XSD');
+goog.require('ol.xml');
+
+
+
+/**
+ * @classdesc
+ * Format for reading WMTS capabilities data
+ *
+ * @constructor
+ * @extends {ol.format.XML}
+ * @api
+ */
+ol.format.WMTSCapabilities = function() {
+  goog.base(this);
+
+  /**
+   * @type {string|undefined}
+   */
+  this.version = undefined;
+  this.owsParser = new ol.format.OWS();
```

Comment:
You can also make this one private I guess, and provide a type for it. I'd also change its name to `owsFormat_`.

```js
  /**
   * @type {ol.format.OWS}
   * @private
   */
  this.owsFormat_ = new ol.format.OWS();
```